packages/cli/src/Init.ts

```diff
@@ -327,6 +339,77 @@ export class Init implements Command {
     const previewFeatures = args['--preview-feature']
     const output = args['--output']
 
+    if (datasourceProvider === `prisma+postgres`) {
+      /**
+       * Flow:
+       * 1. Check authentication status
+       * 2. If not authed, run login or signup
+       * 3. Find default organization
+       * 4. Create project
+       * 5. Create environment with PPG input
+       * 6. Start polling environment status
+       * 7. Create schema.prisma with PPG data source provider
+       * 8. Create .env.prisma file with PPG URL
+       * 9. Print success message
+       */
+      const PlatformCommands = await import(`./platform/_`)
+
+      const credentials = await credentialsFile.load()
+      if (isError(credentials)) throw credentials
+
+      // TODO: Add interactive login/signup prompt
+      if (!credentials) {
+        const authenticationResult = await PlatformCommands.loginOrSignup()
+        console.log(authenticationResult.message)
+      }
+
+      const platformToken = await PlatformCommands.getTokenOrThrow(args)
+
+      const defaultWorkspace = await PlatformCommands.Workspace.getDefaultWorkspaceOrThrow({ token: platformToken })
+
+      console.log('Creating a project and provisioning a Prisma Postgres© instance...')
```

Comment:
`®` or `™` if we're going to put a thing here, I don't think it's a copyright 😅

Reply:
I noticed this during a huddle as well. Just updated.